db/migrate/20230729221447_add_status_to_exercise_sets.rb

```diff
@@ -0,0 +1,25 @@
+class AddStatusToExerciseSets < ActiveRecord::Migration[7.0]
+  def up
+    add_column :exercise_sets, :status, :string, default: "incomplete"
+
+    ExerciseSet.reset_column_information
```

Comment:
Just a FYI: I don't think you need this. So far as I can tell, `reset_column_information` is only used if your application is actively running. I've never seen happen in a rails app (or any app for that matter).